aiida_vasp/io/parser.py

```diff
@@ -30,6 +30,54 @@ def splitlines(cls, fobj_or_str, d_type=float):
         return [cls.line(l, d_type) for l in lines]
 
 
+class BaseFileParser(BaseParser):
```

Comment:
Please add unit tests for this class in aiida_vasp/io/tests/test_base_file_parser.py

Since it is Abstract, create a minimal subclass to test. Also please make sure to not retest this fuctionality in the other subclasses.